cdap-ui/app/cdap/components/ConfigurationGroup/WidgetWrapper/index.tsx

```diff
@@ -134,7 +134,7 @@ const WidgetWrapperView: React.FC<IWidgetWrapperProps> = ({
       onBlur={onBlur}
     >
       <If condition={!hideWrapper}>
-        <div className={classes.label}>
+        <div className={`widget-wrapper-label ${classes.label}`}>
```

Comment:
use classnames for this to be conistent?

Reply:
classnames not needed since these values should always be consistent, no conditional logic here for the classNames